crates/credentials/Cargo.toml

```diff
@@ -8,9 +8,9 @@ license-file.workspace = true
 
 [dependencies]
 jsonschema = "0.17.1"
-serde = { version = "1.0.193", features = ["derive"] }
-serde_json = "1.0.108"
-serde_with = "3.4.0"
+serde = { workspace = true }
+serde_json = { workspace = true }
+serde_with = { workspace = true }
```

Comment:
the `credentials` and `dids` crate both now depend on serde crates. Moved them to the workspace `Cargo.toml` file so that their versions are always in sync with each other.